### ISSUE TITLE
Add External Module documentation

### ISSUE THREAD

**Issue by pbchase** [redcap-tools/redcap-tools.github.io#59]
VUMC has moved their official external module documentation back into a public github repository at https://github.com/vanderbilt-redcap/external-module-framework-docs. @mmcev106 is leading the project. His plan is to move increasing amounts of the developer documentation out of the REDCap Core code and into this repository. Would this repo be an appropriate addition to the Resources section?

Related to this, UF's [REDCap External Module Development Guide](https://ctsit.github.io/redcap_external_module_development_guide/) might also be a suitable addition for the same section. Mark has adopted about 1/3 of our content into new public VUMC docs so that it can be rev'd along with changes to the EM framework. We'll soon excise that part of the our EM docs, but we intended to keep the rest of the site as a resource both for us and for others who need to build EM development teams

**Comment by wibeasley** [comment]
@pbchase, makes sense to me.  If a 3+ entries accumulate, maybe we should add a dedicated table to EM material.  

side note: I know you're not proposing this, but I want to get it on paper.  I think we should point to resources _about_ EMs, and not to specific EMs themselves.  I don't want to replicate the EM Module Manager that's built into REDCap.  That would be a lot of time and always be out of date.
